assets/react/controllers/Incident/OrderDetails.jsx

```diff
@@ -0,0 +1,167 @@
+  }
+
+  return header(`${t("TASK")} #${task.id}`);
+}
+
+function CustomerShowName({ customer }) {
+  const { t } = useTranslation();
+  let customerName = customer?.username;
+  if (customer?.fullName != null) {
+    customerName = customer.fullName;
+  }
+  return (
+    <p title={customer?.username}>
+      {t("CUSTOMER")}
+      <span>{customerName}</span>
+    </p>
+  );
+}
+
+function showAdjustment(adjustments, adjustmentType) {
+  const total = adjustments[adjustmentType].reduce(
+    (total, adjustment) => total + adjustment.amount,
+    0,
+  );
+  if (total === 0) {
+    return;
+  }
+  return adjustments[adjustmentType].map((adjustment) => (
+    <p key={adjustment.id}>
+      {adjustment.label}
+      <span>{money(adjustment.amount)}</span>
+    </p>
+  ));
+}
+
+function ShowOrderDetails({ order }) {
+  const { t } = useTranslation();
+  return (
+    <>
+      <h5>{t("ORDER_DETAILS")}</h5>
+      <p>
+        {t("SUBTOTAL")}
+        <span>{money(order.itemsTotal)}</span>
+      </p>
+      {showAdjustment(order.adjustments, "delivery")}
+      {showAdjustment(order.adjustments, "tax")}
+      {showAdjustment(order.adjustments, "incident")}
+      <p>
+        {t("TOTAL")}
+        <span>{money(order.total)}</span>
+      </p>
+      <hr />
+    </>
+  );
+}
+
+function ShowCustomerDetails({ customer }) {
+  const { t } = useTranslation();
+  const link = window.Routing.generate("admin_user_edit", {
+    username: customer?.username,
+  });
+  return (
+    <>
+      <h5 style={{ lineHeight: "24px" }}>
+        {t("CUSTOMER_DETAILS")}
+        {_externalLink(link)}
+      </h5>
+      <CustomerShowName customer={customer} />
+      {customer?.email && (
+        <p>
+          {t("EMAIL")}
+          <span>{customer?.email}</span>
+        </p>
+      )}
+      {customer?.telephone && (
+        <p>
+          {t("PHONE")}
+          <span>{customer?.telephone}</span>
+        </p>
+      )}
+    </>
+  );
+}
+
+export default function ({ delivery }) {
+  delivery = JSON.parse(delivery);
+  const { t } = useTranslation();
+  const { loaded, order, incident } = store.getState();
+  const { task } = incident;
+
+  if (!loaded) {
+    return null;
+  }
+
+  return (
+    <div className="order-details-card">
+      <Heading task={task} delivery={delivery} order={order} />
+      <p className="text-muted">
+        {t("DATE")}: {formatTime(task)}
+      </p>
+      <hr />
+      {order && <ShowOrderDetails order={order} />}
+      <h5>
+        <span style={{ textTransform: "capitalize" }}>
+          {task.type.toLowerCase()}
+        </span>{" "}
+        details
+      </h5>
+      <p>{task.address.name}</p>
+      <p>{task.address.streetAddress}</p>
+      <p>{task.address.telephone}</p>
+      {task.weight && <p>{weight(task.weight)}</p>}
+      <div className="mt-3">{<TaskStatusBadge task={task} />}</div>
+      <hr />
+      {order?.customer && <ShowCustomerDetails customer={order.customer} />}
+    </div>
+  );
```

Comment:
I think the word `show` is unnecessary here, it could be simply:
CustomerShowName > CustomerName
ShowOrderDetails -> OrderDetails
ShowCustomerDetails -> CustomerDetails

Reply:
Haha, yeah, there's this internal private function that gets converted on the fly into a component, and nobody bothers renaming it 🫠
Nice catch +1